tests/security/with-api-prefix/account.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test account endpoint.
+
+chapters:
+  - synopsis: Get account details.
+    path: /_plugins/_security/api/account
+    method: GET
+    response:
+      status: 200
+  # TODO: following test can be changed to interact with test user, to be created in prologues, once https://github.com/opensearch-project/opensearch-api-specification/issues/438 is implemented
```

Comment:
This was implemented, try it!

Reply:
Currently, the only way to pass credentials other than default is setting them via command-line. Since the requirement here is only test-specific, it would not work. 
Filed an issue: https://github.com/opensearch-project/opensearch-api-specification/issues/485

Reply:
So as written is the test correct? 

Change the TODO to be more directed, maybe say "After #xyz is fixed, add ...."